checkov/cdk/checks/python/ALBDropHttpHeaders.yaml

```diff
@@ -0,0 +1,16 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_131
+  name: Ensure that ALB drops HTTP headers
+  category: NETWORKING
+scope:
+  languages:
+    - python
+definition:
+  pattern: aws_cdk.aws_elasticloadbalancingv2.CfnLoadBalancer(<ANY>)
+  conditions:
+    - not_pattern: |
+        aws_cdk.aws_elasticloadbalancingv2.CfnLoadBalancer(<ANY>, type='application' ,<ANY>, load_balancer_attributes=[<ANY>, {'key': 'routing.http.drop_invalid_header_fields.enabled','value': 'true'} ,<ANY>] ,<ANY>)
```

Comment:
I think here we also need to add all the combinations possible

Reply:
Done